### DISCUSSION TITLE
¡Bienvenid@as al foro del curso Design Thinking (1ra Gen) con Patrones Hermosos!

### DISCUSSION THREAD

**Discussion by KasturiSShah** [PatronesHermosos/design-thinking-1ra-generacion#1] [Anuncios]
Esperamos que:

- Compartes tus ideas
- Actualices tu progreso con las tareas semanales
- Aclares tus dudas aquí
- Comuniques con otros miembros de la comunidad
- Dar la bienvenida a los demás y ser de mente abierta. Recuerda que esta es una comunidad que construimos junt@s 💪.

Recordatorio: Por usar este repositorio, estamos tod@s comrpometid@s al Código de Conducta de Patrones Hermosos y a las pautas de Integridad Académica de Patrones Hermosos.

Para comenzar, comenta a continuación: ¿cual es tu tipo de helado favorito? 😁

**Comment by PIERRECAR** [comment]
Buenas noches, mi helado favorito es el de vainilla.

**Comment by Gaga1982** [comment]
Hola gente. mi helado favorito es el de frutos del bosque (moras, grosellas, arándanos, frambuesas) Y les cuento que en mi país (ARGENTINA) venden el helado por kilo y hay gustos inimaginables!

**Comment by MaryAnnHernani** [comment]
Buen día mi helado favorito es el de papaya arequipeña

**Reply by PilyRmz** [reply]
:O nunca había escuchado ese sabor de helado

**Comment by EdittTB** [comment]
Buenas tardes, mi helado favorito es cookies & cream!

**Comment by Neri-hV** [comment]
mi helado favorito es el de café y en estos momentos no puedo comerlo :(

**Comment by avrx4** [comment]
¡Mucho gusto! No pude asistir a la 1.ª reunión y no sé con qué grupo estoy. Podrían ayudarme por fa

**Reply by KasturiSShah** [reply]
Hola @avrx4 
Mucho gusto, y bienvenida!
¿Quieres completar el proyecto sola o en equipo? Para poder asignarte a un equipo personal o grupal. (Uno de los factores sería la disponibilidad de tiempo que tienes para coordinar con otra gente, entre otros.)
¿Podrías enviar un correo a preguntas@patronas.org con tu preferencia?
Gracias
Kasturi

**Reply by ayekn** [reply]
Buenas si querés podés hacerlo conmigo! Yo perdí mi grupo

**Comment by PilyRmz** [comment]
Hola, estaba pensando que no tengo un sabor de helado favorito porque me gustan muchos sabores, y siempre me tardo mil años en elegir jeje. Pero casi siempre es coco, queso, choco chips, mamey, yogurt, mandarina.... no sé. Creo q más podría decir que lo que más me gusta frío son los frappés

**Comment by AmbarPranahuti** [comment]
Holaaa! Una disculpa no pude entrar a la sesión de apertura, ¿me podrían asignar un equipo? :c

**Reply by KasturiSShah** [reply]
@AmbarPranahuti te asigno a un equipo ahora según los correos que intercambiamos!
